packages/dosespot-react/src/common.ts

```diff
@@ -0,0 +1,21 @@
+import { Identifier } from '@medplum/fhirtypes';
+
+export const MEDPLUM_BOT_SYSTEM = 'https://www.medplum.com/bots';
```

Comment:
nit: in the health gorilla integration, I tried to use a system string that was more specificy that this is a medplum integration: `https://www.medplum.com/integrations/bot-identifier`

Might be overkill, but might be nice to be consistent across our integration

https://github.com/medplum/medplum-ee/blob/19c61e15a21f646d957cf2c2c3a1a404212bc64c/packages/health-gorilla/src/common/constants.ts#L5